syntax = "proto2";

package rv.protos;

message ClassItem {
    required int32 id = 1;
    required string name = 2;
    optional string color = 3;
}
